vrpn_Imager.h

```diff
@@ -532,7 +532,7 @@ class VRPN_API vrpn_Imager_Region {
                 // varying slowest.  Depth steps are therefore the largest
                 // steps.
                 val =
-                    ((vrpn_uint16 *)
+                    ((const vrpn_uint16 *)
```

Comment:
It amazes me that this code apparently works, since it looks like some pretty solid violation of strict aliasing rules...